app/views/projects/versions/create.scala.html

```diff
@@ -54,7 +56,11 @@ <h3 class="panel-title">
                                                 @version.description.map { description =>
                                                   @description
                                                 }.getOrElse {
-                                                  <i class="minor">@messages("version.create.noDescription")</i>
+                                                    @project.description.map { description =>
+                                                        @description
```

Comment:
Does this map do anything?

Reply:
Yeah this map is needed because its an `Option[String]` 
So first it will check if you have a version description if not it will fallback to project description or else it will be no description

The reason I changed it like this is because you can edit the message during the upload. But changing it will add it to the project and not to the version, so if you originally upload a plugin with no description the `version.description` is empty and if you change it it will get populated in `project.description`

Reply:
But the map itself isn't doing anything to the variable. `f(x) = x` doesn't do anything.